sql/moz-fx-data-shared-prod/org_mozilla_fenix_derived/nimbus_recorded_targeting_context_v1/metadata.yaml

```diff
@@ -0,0 +1,15 @@
+friendly_name: Nimbus Recorded Targeting Context (Fenix)
+description: |-
+  A query to obtain the previous day's recorded Nimbus targeting context values for each client.
+owners:
+- chumphreys@mozilla.com
+labels:
+  incremental: true
+scheduling:
+  dag_name: bqetl_default
```

Comment:
It might make sense to include these in the `bqetl_experiments_daily` DAG, which queries for things related to experimentation like enrollment and search events.

Otherwise, we'd probably want to create a new DAG for this.

@scholtzan What do you think?

Reply:
That makes sense to me